### ISSUE TITLE
Incongruence with some features

### ISSUE THREAD

**Issue by TenenteColombian** [Radiomics/pyradiomics#648]
**Describe the bug**
I am currently working on my MSc thesis in Mathematics, and I’m fairly new with PyRadiomics. I’m having some issues when extracting some features. I was comparing their output with the ones extracted using another software that was suggested to me (LIFEx), and I found some inconsistencies with the following: for the GLRLM Features

- LGLRE,
- HGLRE,
- SRLGLE,
- SRHGLE,
- LRLGLRE,
- LRHGLRE,

and for the GLSZM Features

- LGLZE,
- HGLZE,
- SALGLE,
- SAHGLE,
- LALGLE,
- LAHGLE.

All the other ones were just fine, they coincided. I wondered if, perhaps, the formulas used were the same, but they seemed so. I can’t understand if I’m missing some obvious parameters’ settings, but again, I’m just starting using PyRadiomics.

Thank you in advance for anyone who can help me.

**Version (please complete the following information):**
 - OS: MacOS
 - Python version: 3.8.5
 - PyRadiomics version: 3.0

**Comment by JoostJM** [comment]
PyRadiomics was part of the forming of IBSI and as such, we conducted experiments to see to what extent PyRadiomics overlaps with the definitions mentioned there, and what the cause was for any differences.

You can find some answers in #498 and in the documentation [here](https://pyradiomics.readthedocs.io/en/latest/faq.html#does-pyradiomics-adhere-to-ibsi-definitions-of-features). In short, there are some differences, but not to the extent that either PyRadiomics or IBSI could be considered "wrong", just "different".

@fedorov, @pieper, would it be an idea to (at least optionally) allow users to use IBSI-like resampling? And optional "IBSI-correction" for kurtosis? I'm not terribly convinced of using their definition of bin-edge definition (using the lower edge of the resegmentation range), as this can cause errors when applying filters (which may have a completely different range of intensities).

Finally, I don't know if this explains all the differences you are seeing when comparing to LIFEx, but any differences that remain may be best explained by asking the LIFEx team if they have any discrepancies with the IBSI.

**Comment by pieper** [comment]
@JoostJM could be a good feature to have ISBI-style computations.

Cross-linking to the same question posted on slicer discourse:

https://discourse.slicer.org/t/incongruence-with-some-feautures/13864

**Comment by fedorov** [comment]
I agree it would be helpful to the users to have a way to configure to use IBSI-compliant computation. It is really unfortunate that although pyradiomics is probably the most prominent implementation that was evaluated by IBSI, the agreement was not reached to use consistent definitions between IBSI and pyradiomics.

**Comment by TenenteColombian** [comment]
Thank you very much for your answers!
